dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/Users.cshtml

```diff
@@ -0,0 +1,110 @@
+@page "/admin/users"
+@model TeacherIdentity.AuthServer.Pages.Admin.UsersModel
+@{
+    ViewBag.Title = "Users";
+}
+
+<h1 class="govuk-heading-xl">@ViewBag.Title</h1>
+
+<govuk-tabs>
+    <govuk-tabs-item label="Open" id="open-tab">
+        <p class="govuk-heading-s ">These records need to be checked against the DQT.</p>
+
+        <p>Showing @Model.OpenTotal of @Model.OpenTotal total</p>
+
+        @if (Model.Open!.Length > 0)
+        {
+            <table class="govuk-table">
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header">Name</th>
+                        <th scope="col" class="govuk-table__header">Email</th>
+                        <th scope="col" class="govuk-table__header"></th>
+                    </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                    @foreach (var user in Model.Open!)
+                    {
+                        <tr class="govuk-table__row" data-testid="user-@user.UserId">
+                            <td class="govuk-table__cell"><a asp-page="EditUser" asp-route-userId="@user.UserId">@user.Name</a></td>
```

Comment:
Not sure if this refers to a new page, or is supposed to redirect to User page

Reply:
Good spot. This is a hangover from before `EditUser` was renamed to `User`.